public/js/angular/directives.js

```diff
@@ -81,7 +79,8 @@ angular
           });
         }
       };
-    }])
+    }
```

Comment:
not sure, I don't remember making that change, so might have been jsbeautify again.

Although I have made a lot of changes so I'm sure it's possible I forgot.